webview-ui/src/components/settings/ApiOptions.tsx

```diff
@@ -490,6 +490,14 @@ const ApiOptions = ({
 
 			{selectedProvider === "openai-native" && (
 				<>
+					<VSCodeTextField
```

Comment:
Can we make this behind a checkbox like we do for Anthropic, OpenRouter, etc?

Reply:
Okay, I believe it will be more user-friendly.